### ISSUE TITLE
VCF: Phasing and Alt Loci

### ISSUE THREAD

**Issue by adamnovak** [samtools/hts-specs#31]
If I want to call a SNP at human chr6_GL000255v2_alt:1,144,634, and phase my (potentially haploid) call at that SNP for a sample with other calls on the main chr6, how would I express that in VCF? The spec says that phase sets are restricted to things with matching "chromosome" fields, so I can't put both the variants into a phase set, and anyway the first allele in this haploid call might be phased with the second allele in the diploid call on chr6.

Do I need to wire up the appropriate copy of chr6 to the alt locus with a breakend that's phased at both ends?

Basically, the issue is that the spec should specify how to describe variants that fall into the alt loci which are suddenly very common in GRCh38, especially with regard to phasing. Perhaps with an example.

**Comment by d-cameron** [comment]
#421
